bottom/client.py

```diff
@@ -1,14 +1,20 @@
 import asyncio
 import collections
 import functools
+from typing import Any, Callable, DefaultDict, List, Optional  # noqa
```

Comment:
Using `Dict` here is perfectly fine - I will add a commit. `functools.partial` (protocols in general) will probably need new `typing` module, so it may need a new release. However, in this case we aren't defining any new protocols, but using builtin class, so future mypy version will simply use new `typing.pyi` from typeshed and it will be sufficient. Same goes for `AbstractEventLoop` methods: once this is fixed in typeshed repo, it will be merged into mypy and then released on PyPI; no new Python distribution needed.